docs/preview/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Home"
+layout: default
+permalink: /
+redirect_from:
+ - /index.html
+---
+
+[![NuGet Badge](https://buildstats.info/nuget/Arcus.{Component}?includePreReleases=true)](https://www.nuget.org/packages/Arcus.{Component}/)
+
+# Installation
+
+The Arcus.{Component} can be installed via NuGet:
+
+```shell
+PM > Install-Package Arcus.{Component}
```

Comment:
We should update this

Reply:
See master

Reply:
Merged with master and inserted correct info.